packages/amplify-category-hosting/index.js

```diff
@@ -58,9 +59,20 @@ async function configure(context) {
 
 function publish(context, service, args) {
   const { enabledServices } = categoryManager.getCategoryStatus(context);
-
   if (enabledServices.length > 0) {
     if (enabledServices.includes(service)) {
+      // checks if hosting with S3 and CloudFront and has a secure URL otherwise,
+      // the user is hosting from S3 without CloudFront distribution and should throw an error
+      if (service === 'S3AndCloudFront') {
+        const cfnOutput = context.exeInfo?.amplifyMeta.hosting.S3AndCloudFront.output;
```

Comment:
I may be mistaken, but I think @edwardfoyle was suggesting to change this to:

```suggestion
        const cfnOutput = context.exeInfo?.amplifyMeta?.hosting?.S3AndCloudFront?.output;
```

Reply:
I misunderstood, 

I thought it was 

`context.exeInfo?.amplifyMeta.hosting.S3AndCloudFront.output;` or `context.exeInfo.amplifyMeta.hosting.S3AndCloudFront.output;`